rero_ils/modules/documents/api.py

```diff
@@ -259,6 +259,18 @@ def replace_refs(self):
                 contribution, online = Contribution.get_record_by_ref(ref)
                 if contribution:
                     contributions[idx]['agent'] = contribution
+        subjects = self.get('subjects', [])
+        for idx, subject in enumerate(subjects):
+            subject_ref = subject.get('$ref')
+            subject_type = subject.get('type')
+            if subject_ref:
+                if subject_type in ['bf:Person', 'bf:Organisation']:
+                    subject, online = Contribution.get_record_by_ref(
```

Comment:
if `online` is never used, it sounds better to use `subject, _ = Contributioin.get_record_by_ref(...)`